src/cow-react/pages/Account/ens/index.ts

```diff
@@ -5,7 +5,7 @@ import { EnsNamesQuery } from '@cow/pages/Account/ens/types'
 const CHAIN_SUBGRAPH_URL: Record<number, string> = {
   [SupportedChainId.MAINNET]: 'https://api.thegraph.com/subgraphs/name/ensdomains/ens',
   [SupportedChainId.GOERLI]: 'https://api.thegraph.com/subgraphs/name/ensdomains/ensgoerli',
-  [SupportedChainId.RINKEBY]: 'https://api.thegraph.com/subgraphs/name/ensdomains/ensrinkeby',
+  // [SupportedChainId.RINKEBY]: 'https://api.thegraph.com/subgraphs/name/ensdomains/ensrinkeby',
```

Comment:
smae here, you can delete as you did in some other places

Reply:
Deleted everywhere in `cow-react` directory and keep commented in the `custom` dir.